lib/stripe.rb

```diff
@@ -220,10 +220,8 @@ def self.uri_encode(params)
   end
   class << self
     extend Gem::Deprecate
-    deprecate :uri_encode, "Stripe::Util#encode_parameters", 2016, 01
+    deprecate :uri_encode, "Stripe::Util#encode_parameters", 2016, 0o1
   end
 end
 
-unless ENV["STRIPE_LOG"].nil?
-  Stripe.log_level = ENV["STRIPE_LOG"]
-end
+Stripe.log_level = ENV["STRIPE_LOG"] unless ENV["STRIPE_LOG"].nil?
```

Comment:
I guess pulling things onto one line is alright, but seems kind of unnecessary. Probably not worth changing though.